client/src/components/FeedbackForm/FeedbackModal.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Modal, ModalHeader, ModalBody } from 'reactstrap';
+import { func, bool } from 'prop-types';
+import { Link } from 'react-router-dom';
+import FeedbackForm from './FeedbackForm';
+import * as ROUTES from '../../constants/routes';
+import './FeedbackModal.module.scss';
+
+const MODAL_PROPTYPES = {
+  isOpen: bool.isRequired,
+  toggle: func.isRequired
+};
+
+const FeedbackModal = React.memo(({ isOpen, toggle }) => {
+  const authenticatedUser = useSelector(state => state.authenticatedUser.user);
+
+  return (
+    <Modal styleName="feedback-modal" isOpen={isOpen} toggle={() => toggle()}>
+      <ModalHeader toggle={toggle} charCode="x">
```

Comment:
Can you style this header like the header for the ticket creation modal? Grey header, etc